TarkovMapOverlay/MainWindow.xaml.cs

```diff
@@ -536,5 +537,25 @@ void SourceChangedHandler(object sender, EventArgs e)
         {
             image.RenderTransform = new MatrixTransform();
         }
+
+        private void MenuItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            setting_transparency.IsChecked = true;
+            opacity = sliderMenu.Value;
+            this.Background = new SolidColorBrush(Colors.Black) { Opacity = 0 };
+            this.Opacity = opacity * 0.01;
+            this.BorderBrush = new SolidColorBrush(Colors.Black) { Opacity = 0 };
+            transparentBackground = true;
```

Comment:
I can double click the button to make the window fullscreen or small again but why does the background go transparent also. Why would someone want to have this functionality of doubleclicking to go full screen? Why not make it a menu item?

Reply:
this is indeed a good question. I thought it would maybe look nicer if the menubutton is nice at the upper left corner of the screen while the "downscaled" Image can stay in the lower right corner for example. if so, the background would get in the way if you decide to maximize the whole window with the menubutton while the background is still opaque.

We don't have to do this, it was just my personal preference to have the menubutton out of the way while i can leave the image open ingame.

Reply:
and yes you're absolutely right, it could be better making another menuitem for that.

Reply:
Alright, I think it's ok but I don't think it should be done when making the window small again.

Have you actually tried using this because I think the window will catch all mouse input and prevent you from playing?

Reply:
Just discard my changes, let's leave the prog. as is^^

Reply:
the transparent background didn't prevent, but the image does

Reply:
let's discard, you're right. doesn't make sense at all

Reply:
Good idea, maybe we can use it if we can prevent the mouse interaction 😄

Reply:
the interaction with the image itself?

Reply:
haha yea

Reply:
i just see one problem there, if all controls become click-trought, how can one interact with the menu?

Reply:
yea, either the menu is left clickable or we need a hotkey to toggle it. Either is ok for me.